chromium_src/chrome/browser/privacy_sandbox/privacy_sandbox_service_unittest.cc

```diff
@@ -139,6 +159,10 @@ class PrivacySandboxServiceTest : public testing::Test {
 
   std::unique_ptr<PrivacySandboxService> privacy_sandbox_service_;
   browsing_topics::MockBrowsingTopicsService* mock_browsing_topics_service_;
+  first_party_sets::FirstPartySetsPolicyService
```

Comment:
not sure I'm following what's going on in these tests. The privacy sandbox is disabled, right? cc @pes10k

Reply:
Yes, privacy sandbox is disabled in https://github.com/brave/brave-core/blob/master/chromium_src/content/public/common/content_features.cc#L21 and in https://github.com/brave/brave-core/blob/master/chromium_src/components/privacy_sandbox/privacy_sandbox_features.cc

Reply:
so if fps is also disabled, what is this file doing? I don't even see any actual tests here

Reply:
Hmm, it seems Mario removed the tests as no longer needed in [Chromium 101 bump](https://github.com/brave/brave-core/commit/383544c8a52e150997076600bacc151b873d4217), but left the file around for some reason. We can just remove it, I think.

Reply:
Removed in c0dc9729faa983e1f51f6f2c1510295694284e51